src/databricks/labs/ucx/hive_metastore/locations.py

```diff
@@ -31,6 +37,87 @@ class Mount:
     source: str
 
 
+# Union with string is invalid syntax: "LocationTrie" | None
+OptionalLocationTrie = Optional["LocationTrie"]  # pylint: disable=consider-alternative-union-syntax
+
+
+@dataclass
+class LocationTrie:
+    """A trie datastructure to search locations.
+
+    Used to find overlapping locations.
+
+    TODO
+    ----
+    - [ ] : Include external locations pointing to SQL sources
```

Comment:
it's probably `wontfix`

Reply:
Removed, kept issue to decide what to do with it